### PR TITLE
[SCB-1147] Log slow invocation stage trace information

### PR DESCRIPTION
**Pull request by wujimin**
- support global/service/schema/operation level configuration
- support enabled and time configuration
- enabled default to be false
- time default to be 1000 millisecond

### CODE REVIEW

File: core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  public static final List<String> PRODUCER_OP_PRIORITY = Arrays.asList(
+      ".${schema}.${operation}",
+      ".${schema}",
+      "");
+
+  @InjectProperty(keys = "${consumer-producer}.invocation.slow.enabled${op-priority}", defaultValue = "false")
+  private boolean slowInvocationEnabled;
+
+  @InjectProperty(keys = {"${consumer-producer}.invocation.slow.msTime${op-priority}"}, defaultValue = "1000")
```
Comment (yhs0092) [inline core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java]: `msTime${op-priority}`
Should there be a dot between `msTime` and `${op-priority}`?
The same question to line 48.

File: core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  public static final List<String> CONSUMER_OP_ANY_RIORITY = Arrays.asList(
+      "${service}.${schema}.${operation}",
+      "${service}.${schema}",
+      "${service}");
+
+  public static final List<String> PRODUCER_OP_ANY_PRIORITY = Arrays.asList(
+      "${schema}.${operation}",
+      "${schema}");
+
+  public static final List<String> CONSUMER_OP_RIORITY = Arrays.asList(
```
Comment (yhs0092) [inline core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java]: There seems a typo. Maybe it should be `CONSUMER_OP_PRIORITY`?
The same problem to line 28

File: core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java (hunk 1 above)
Comment (wujimin) [reply core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java]: op-priority reference to CONSUMER_OP_RIORITY or PRODUCER_OP_PRIORITY
already have prefix dot

File: core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java (hunk 2 above)
Comment (wujimin) [reply core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java]: done

File: core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java (hunk 1 above)
Comment (yhs0092) [reply core/src/main/java/org/apache/servicecomb/core/definition/OperationConfig.java]: OK, got it

**Pull request merged**